RELEASE/scripts/autoscend/auto_choice_adv.ash

```diff
@@ -317,7 +317,42 @@ boolean auto_run_choice(int choice, string page)
 			break;
 		case 604: // Welcome to the Great Overlook Lodge (Twin Peak Part 1)
 		case 605: // Welcome to the Great Overlook Lodge (Twin Peak Part 2)
+			run_choice(1); // always advance to next option via choice 1
+			break;
 		case 606: // Lost in the Great Overlook
+			if(in_bhy())
+			{
+				// we can't make an oil jar to solve the quest, just adventure until the hotel is burned down
+				run_choice(6); // and flee the music NC
+				break;
+			}
+			// do init if we can
+			if(options contains 4)
+			{
+				run_choice(4);
+				break;
+			}
+			// do oil jar if we can
+			if(options contains 3)

```

Comment:
Alright, had it happen again this run. Probably should add a check for oil jar here because it is just going into this choice regardless of whether we have one or not.
[twin peak.txt](https://github.com/loathers/autoscend/files/14512584/twin.peak.txt)

Reply:
```suggestion
			if(options contains 3 && item_amount($item[oil jar]) > 0)
```